bin/Cores/cores.json

```diff
@@ -118,6 +118,11 @@
     "name": "ProSystem",
     "systems": [51]
   },
+  "mednafen_wswan_libretro":{
+    "name": "Beetle Cygne",
```

Comment:
This should be "Mednafen WonderSwan" to match the string displayed in the title bar.